.gitignore

```diff
@@ -48,3 +48,4 @@ regtests/chdman/temp
 regtests/jedutil/output
 /CMakeLists.txt
 /samples
+/3rdparty
```

Comment:
This shouldn’t be changed.

Reply:
Corrected. I'm so sorry for the failures, I'm a newbie at this, I'm learning